README.md

```diff
@@ -107,7 +107,7 @@ $ ./sourcery --sources <sources path> --templates <templates path> --output <out
 
 - `--sources` - Path to a source swift files or directories. You can provide multiple paths using multiple `--sources` option.
 - `--templates` - Path to templates. File or Directory. You can provide multiple paths using multiple `--templates` options.
-- `--force-parse` - File extensions of Sourcery generated file you want to parse. You can provide multiple extension using multiple `--force-parse` options. (i.e. `file.toparse.swift` will be parsed even if generated by Sourcery if `--force-parse toparse`). Useful when trying to implement a multiple phases generation.
+- `--force-parse` - File extensions of Sourcery generated file you want to parse. You can provide multiple extension using multiple `--force-parse` options. (i.e. `file.toparse.swift` will be parsed even if generated by Sourcery if `--force-parse toparse`). Useful when trying to implement a multiple phases generation. Any inline code with annotation ending with "." and this option would also be parsed(i.e. inline code beginning with // sourcery:inline:auto:Command.MirrorEnum would be parsed even if generated by Sourcery if `--force-parse MirrorEnum`).
```

Comment:
Can we rephrase it? I don't get what "ending with ." means from this description and tbh I find original description quite confusing as well.

```suggestion
- `--force-parse` - normally Sourcery does not process files or code within inline annotations generated by Sourcery itself. To force parsing of such files or code within inline annotations pass the extension (without `.swift`) of the file or the name of the annotation to this option. I.e. to process file `*.generated.swift` you can use `--force-parse generated`, to process code within `sourcery:inline:auto:Type.AutoCodable` annotation you can use `--force-parse AutoCodable`. You can provide this option multiple times. Useful when trying to implement a multiple phases generation.
```

@krzysztofzablocki does this describe it correctly? I'm not sure I understand how ti works on files.
I'm also not sure using one option for both files and annotations is a good idea.

Reply:
I think re-using same argument name for both files and inline blocks is fine, it's rare to use feature anyway